rs/nns/governance/src/proposals/update_canister_settings.rs

```diff
@@ -75,6 +76,7 @@ impl UpdateCanisterSettings {
         let memory_allocation = settings.memory_allocation.map(Nat::from);
         let freezing_threshold = settings.freezing_threshold.map(Nat::from);
         let wasm_memory_limit = settings.wasm_memory_limit.map(Nat::from);
+        let _wasm_memory_threshold = settings.wasm_memory_threshold.map(Nat::from);
```

Comment:
Should we add this field to RootCanisterSettings?

Reply:
Yes, but it could be done in a follow-up PR ideally

Reply:
Thank you. I added it manually. I also assumed that the generator would add fields only inside `/gen/` files, while I manually added them to the rest.